src/cmakeFindModules/OpenMVGConfig.cmake.in

```diff
@@ -52,6 +52,14 @@ MACRO(OPENMVG_REPORT_NOT_FOUND REASON_MSG)
   RETURN()
 ENDMACRO(OPENMVG_REPORT_NOT_FOUND)
 
+# whether it is built with CCTags
+set(OpenMVG_USE_CCTAG @OpenMVG_USE_CCTAG@)
+
+if(OpenMVG_USE_CCTAG)
+  set(CCTag_DIR @CCTag_DIR@)
```

Comment:
to make it more portable maybe here we can put
```cmake
if(NOT CCTag_DIR)
  set(CCTag_DIR @CCTag_DIR@)
```
this way if a CCTag_DIR is defined in the project using openMVG that version of CCTag can be used instead of the one it was compiled with openMVG can be used. It should improve the portability. But boh...